packages/strategy-tests/src/lib.rs

```diff
@@ -566,11 +569,31 @@ impl Strategy {
                         document_type,
                         contract,
                     }) => {
+                        // Get the first 10 identities who are eligible to submit documents for this contract
+                        let first_10_eligible_identities: Vec<Identity> = current_identities
+                            .iter()
+                            .filter(|identity| {
+                                mempool_document_counter
+                                    .get(&(identity.id(), contract.id()))
+                                    .unwrap_or(&0)
+                                    < &24u64
+                            })
+                            .take(10)
+                            .cloned()
```

Comment:
Why do you clone them? I see you passing reference below anyways

Reply:
a value of type `Vec<dpp::identity::Identity>` cannot be built from an iterator over elements of type `&dpp::identity::Identity`